cli/src/main/java/org/infinispan/cli/commands/Schema.java

```diff
@@ -43,10 +45,6 @@ public CommandResult exec(ContextAwareCommandInvocation invocation) {
       } else if ((upload == null) && (args.size() != 2)) {
          throw Messages.MSG.illegalCommandArguments();
       }
-      CommandInputLine cmd = new CommandInputLine(Schema.CMD)
-            .arg(KEY, args.get(0))
-            .optionalArg(VALUE, args.size() > 1 ? args.get(1) : null)
-            .optionalArg(FILE, upload != null ? upload.getAbsolutePath() : null);
-      return invocation.execute(cmd);
+      return invocation.call(c -> upload == null ? c.schemas().put(args.get(0), args.get(1)) : c.schemas().put(args.get(0), RestEntity.create(MediaType.TEXT_PLAIN, new File(upload.getAbsolutePath()))));
```

Comment:
same as put, wouldn't be better to have an argument for the schema name and another for the content?
is there a case where a file isn't used? I doubt people will write the schema on hand inside the cli prompt.

Reply:
This PR is not about changing the commands :)